### PR TITLE
removing languages and asg for private profile

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

File: src/Containers/Dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -52,8 +49,8 @@ class DashboardContainer extends Component {
         clientClassifications={userClassifications}
         userClassificationsHasErrored={userClassificationsHasErrored}
         showClassifications
-        showLanguages={showLanguages()}
-        showAssignments={showAssignments()}
+        showLanguages={false}
+        showAssignments={false}
```
Comment (scott062) [inline src/Containers/Dashboard/Dashboard.jsx:53]: 😮

**Review by scott062 — APPROVED** [review]

**Comment by elizabeth-jimenez** [comment]
Confirmed Leah's private profile has no languages or assignment history - Looking at a client's Leah sees both modules.

**Comment by elizabeth-jimenez** [comment]
Jenny Private - no languages, no asg

**Comment by elizabeth-jimenez** [comment]
wendy private - no lang, no asg. Wendy seeing public sees languages

**Comment by elizabeth-jimenez** [comment]
preston private - no lang, no asg. Public: languages

**Comment by elizabeth-jimenez** [comment]
abigail private: no lang, no asg. Public: lang

**Pull request merged**